examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -110,8 +112,28 @@ def save_model_card(
     repo_folder=None,
     vae_path=None,
 ):
-    img_str = "widget:\n"
+    # tags for the model card
     lora = "lora" if not use_dora else "dora"
+    base_model_tag = "playground" if "playground" in base_model else "stable-diffusion-xl"
+    tags_str = f"""
+tags:
+- {base_model_tag}
+- {base_model_tag}-diffusers
+- diffusers-training
+- text-to-image
+- diffusers
+- {lora}
+- template:sd-lora
+    """
```

Comment:
Might be nice to also update the model card generation function using the utilities used in the other scripts (can take place in another PR too).

Reply:
Yeah for sure! will do in a separate PR